### PR TITLE
fix anonymous in report email

### LINKED ISSUE #1881: The anonymous report is always "No" in analyst email even the anonymous report is submitted
## Summary

The anonymous report field in analyst email is always set to "No" even the anonymous report is submitted. In database, it said
"anonymous" : {
		"anonymousOptions" : [
			"anonymousPage.no",
			"anonymousPage.yes"
		]
	},

## Steps to reproduce

Submit an anonymous report

ano

> How exactly can the bug be reproduced? Be very specific.

## Unresolved questions

> Are there any related issues you consider out of scope for this issue that could be addressed in the future?

### PR DESCRIPTION
**Pull request by sastels**
Fixes #1881 

# Description

Remove the anonymous.no default which was causing a bug in the report email

To test: run locally with the environment variable `MAIL_LOCAL` set to your email address

# Checklist:

- [x] I have looked at my code on GitHub and it all looks good (ex: no random commented out code or console.logs)
- [x] I have added and needed tests for my changes (in particular for new screens)
- [x] I have added a comment to any confusing code
- [x] I have added documentation to any modified front-end code. (Or added missing documentation)

### CODE REVIEW

File: f2/src/forms/defaultValues.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ const formDefaults = {
   prodVersion: '',
   appVersion: '',
   consent: { consentOptions: [] },
-  anonymous: { anonymousOptions: ['anonymousPage.no'] },
```
Comment (DianeLiu2019) [inline f2/src/forms/defaultValues.js:6]: I think you changed default value  to "no" for load testing

Comment (sastels) [reply f2/src/forms/defaultValues.js:6]: That is a good question 😅  - I just checked and the load testing is still fine.

Comment (DianeLiu2019) [reply f2/src/forms/defaultValues.js:6]: good to go.

Comment (DianeLiu2019) [reply f2/src/forms/defaultValues.js:6]: Excellent.

Comment (DianeLiu2019) [reply f2/src/forms/defaultValues.js:6]: Good to go

**Pull request merged**